src/pages/billing/Project.js

```diff
@@ -308,7 +319,56 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
               isClearable: false,
               options: _.map(({ displayName, accountName }) => ({ label: displayName, value: accountName }), billingAccounts),
               onChange: ({ value: newAccountName }) => setSelectedBilling(newAccountName)
+            }),
+            div({ style: { marginTop: '1rem' } },
+              ['Note: Changing the billing account for this billing project will clear the workflow spend report configuration.'])
+          ])])
+        ])
+      ]),
+      div({ style: { color: colors.dark(), fontSize: 14, display: 'flex', alignItems: 'center', margin: '0.5rem 0 0 1rem' } }, [
+        span({ style: { flexShrink: 0, fontWeight: 600, fontSize: 14, marginRight: '0.75rem' } }, 'Workflow Spend Report Configuration:'),
+        span({ style: { flexShrink: 0 } }, 'Edit'),
+        h(Link, {
+          tooltip: 'Configure Workflow Spend Reporting',
+          style: { marginLeft: '0.5rem' },
+          onClick: async () => {
+            if (Auth.hasBillingScope()) {
+              setShowSpendReportConfigurationModal(true)
+            } else {
+              await authorizeAndLoadAccounts()
+              setShowSpendReportConfigurationModal(Auth.hasBillingScope())
+            }
+          }
+        }, [icon('edit', { size: 12 })]),
+        showSpendReportConfigurationModal && h(Modal, {
+          title: 'Configure Workflow Spend Reporting',
+          onDismiss: () => setShowSpendReportConfigurationModal(false),
+          okButton: h(ButtonPrimary, {
+            disabled: !selectedDatasetProjectName || !selectedDatasetName,
+            onClick: async () => {
+              setShowSpendReportConfigurationModal(false)
+              await updateSpendConfiguration(projectName, selectedDatasetProjectName, selectedDatasetName)
+            }
+          }, ['Ok'])
+        }, [
+          h(IdContainer, [id => h(Fragment, [
+            h(FormLabel, { htmlFor: id, required: true }, ['Dataset Project Name']),
+            h(TextInput, {
+              id,
+              onChange: setSelectedDatasetProjectName
             })
+          ])]),
+          h(IdContainer, [id => h(Fragment, [
+            h(FormLabel, { htmlFor: id, required: true }, ['Dataset Name']),
+            h(TextInput, {
+              id,
+              onChange: setSelectedDatasetName
+            }),
+            div({ style: { marginTop: '1rem' } }, [
+              ['See '],
+              h(Link, { href: 'https://support.terra.bio/hc/en-us/articles/360037862771', ...Utils.newTabLinkProps }, ['our documentation']),
+              [' for details on configuring workflow spend reporting for billing projects.']
```

Comment:
I'd expect to see errors in the js console from passing arrays here, the strings shouldn't be in separate arrays.:
```js
div([
  'See ',
  h(Link...),
  ' for details'
])
```